langkit/railroad_diagrams.py

```diff
@@ -0,0 +1,98 @@
+from os import path as P
+import pathlib
+from typing import Union
+
+from langkit.parsers import (
+    Defer, List, Opt, Or, Parser, _Extract, _Row, _Token, _Transform
+)
+
+
+def emit_railroad_diagram(parser: Parser) -> None:
+    """
+    Pass to emit railroad diagrams using the railroad_diagrams library.
+
+    Railroads will be emitted in $BUILD/doc/svg.
+    """
+    try:
+        from railroad import (
+            Choice, Diagram, DiagramItem, End, OneOrMore, Optional,
+            Sequence, Skip, Start, ZeroOrMore
+        )
+    except ImportError:
+        return
```

Comment:
This pass is disabled by default, and I think it would be good to have it actually fail if requested while railroad is missing, so I would suggest remove the try/except/return guard.